src/Mono.Android/Xamarin.Android.Net/AndroidClientHandler.cs

```diff
@@ -189,23 +177,23 @@ protected void AssertSelf ()
 			if (!request.RequestUri.IsAbsoluteUri)
 				throw new ArgumentException ("Must represent an absolute URI", "request");
 
-			java_url = new URL (request.RequestUri.ToString ());
-			java_connection = java_url.OpenConnection () as HttpURLConnection;
-			HttpURLConnection httpConnection = await SetupRequestInternal (request, java_connection).ConfigureAwait (false);
-					return await ProcessRequest (request, httpConnection, cancellationToken);
+			var java_url = new URL (request.RequestUri.ToString ());
+			var java_connection = java_url.OpenConnection ();
```

Comment:
Please use the full type instead of `var` here - it helps when viewing the code from outside an IDE